apps/wing-console/console/server/src/router/test.ts

```diff
@@ -138,14 +159,25 @@ export const createTestRouter = () => {
       const testList = await listTests(simulator);
       const result: InternalTestResult[] = [];
       for (const resourcePath of testList) {
-        const response = await runTest(simulator, resourcePath, ctx.logger);
-        result.push(response);
-        testsState.setTest({
-          id: resourcePath,
-          label: getTestName(resourcePath),
-          status: response.error ? "error" : "success",
-          time: response.time,
-        });
+        try {
+          const response = await runTest(simulator, resourcePath, ctx.logger);
```

Comment:
Maybe reuse this logic across the two places where we run the test?

Reply:
removed